category.js

```diff
@@ -0,0 +1,128 @@
+const pathToJsons = "./Jsons/";
+const dropDown = document.getElementById("category");
+const continueBtn = document.getElementById("continue-btn");
+
+function capitalizeCategories(categories) {
+  const array = [];
+  for (let category of categories) {
+    const upperCategory = category[0].toUpperCase() + category.slice(1);
+    array.push(upperCategory);
+  }
+  return array;
```

Comment:
**Suggestion(non-blocking)**: You can try using `map` here too for quicker clarity, although this is also completely fine!